pkg/koordlet/util/system/kidled_util.go

```diff
@@ -134,12 +138,12 @@ func ParseMemoryIdlePageStats(content string) (*ColdPageInfoByKidled, error) {
 
 // boundary is the index of [1,2)  [2,5)  [5,15)  [15,30)  [30,60)  [60,120)  [120,240)  [240,+inf).
 // if boundary is equeal to 3, it will compute sum([5*scan_period_scands,+inf)) of cold page cache
-func (i *ColdPageInfoByKidled) GetColdPageTotalBytes(boundary int) uint64 {
+func (i *ColdPageInfoByKidled) GetColdPageTotalBytes() uint64 {
 	sum := func(nums ...[]uint64) uint64 {
 		var total uint64
 		for _, v := range nums {
 			for i, num := range v {
-				if i >= boundary {
+				if i >= int(kidledColdBoundary.Load()) {
```

Comment:
This load raises performance problems. Why not use `KidledConfig.KidledColdBoundary` instead of a mutexed KidledColdBoundary?